### PR TITLE
Cleanup backup behaviour.

### PR DESCRIPTION
**Pull request by anghelutar**

### CODE REVIEW

File: upstream/solr6/local/92-init-solr.sh
Comment (wimfabri) [inline upstream/solr6/local/92-init-solr.sh:153]: shouldn't this test als include major versions >2 ?

Comment (vierbergenlars) [reply upstream/solr6/local/92-init-solr.sh:153]: Another opinion: can we just always set the `solr.backup.dir` option?

If it is not supported by a version of Solr, it is just ignored, right?
And that is easier to do than a complex condition to check the version number.

File: upstream/solr6/local/92-init-solr.sh
Comment (vierbergenlars) [inline upstream/solr6/local/92-init-solr.sh:157]: Is this so when you have multiple shards in one Solr server, they all write to a different backup directory instead of overwriting eachother?

File: upstream/solr6/local/92-init-solr.sh
Comment (anghelutar) [reply upstream/solr6/local/92-init-solr.sh:153]: > shouldn't this test als include major versions >2 ?

Version 3 is elasticsearch. It is not clear that Solr will get new major versions.

Comment (Ranecdev) [reply upstream/solr6/local/92-init-solr.sh:153]: It is indeed ignored. 

Although I don't really like it to have a property in solrcore.properties that does not function. It might be confusing.

**Comment by anghelutar** [comment]
Since I no longer belong to xenit organization, feel free to remove me as maintainer (maintainer without admin rights makes no sense) and continue as you wish.

File: upstream/solr6/local/92-init-solr.sh
Comment (Ranecdev) [reply upstream/solr6/local/92-init-solr.sh:157]: Yes also needed for the typical alfresco and archive cores

File: upstream/solr6/local/92-init-solr.sh
Comment (vierbergenlars) [inline upstream/solr6/local/92-init-solr.sh:199]: @Ranecdev You forgot to remove the version check here.

Comment (Ranecdev) [reply upstream/solr6/local/92-init-solr.sh]: Done

**Review by vierbergenlars — APPROVED** [review]

**Pull request merged**
